dozer/cogs/moderation.py

```diff
@@ -52,18 +52,17 @@ def __init__(self, bot):
             await target.send(embed=modlog_embed)
         except discord.Forbidden:
             await orig_channel.send("Failed to DM modlog to user")
-        with db.Session() as session:
-            modlog_channel = session.query(GuildModLog).filter_by(id=actor.guild.id).one_or_none()
+        modlog_channel = await GuildModLog.get_by_guild(guild_id=actor.guild.id)
+        if orig_channel is not None:
+            await orig_channel.send(embed=modlog_embed)
+        if len(modlog_channel) != 0:
+            if global_modlog:
+                channel = actor.guild.get_channel(modlog_channel[0].modlog_channel)
+                if channel is not None and channel != orig_channel: # prevent duplicate embeds
+                    await channel.send(embed=modlog_embed)
+        else:
             if orig_channel is not None:
```

Comment:
the `orig_channel` check was already done on line 56. Replace line 58 with a `elif`?

Reply:
No, these are separate operations. One always needs to happen, while the other needs to happen only sometimes.